src/main/java/aluczak/reservationsystem/MovieController.java

```diff
@@ -0,0 +1,82 @@
+package aluczak.reservationsystem;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.util.Pair;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class MovieController {
+    @Autowired
+    MovieRestRepository movieRestRepository;
+
+    @GetMapping("/movies/byScreeningTimeResponse")
+    public List<MovieDTO> getMovieFromScreeningTime(
+            @RequestParam("startDateTime")
+            @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime startDateTime,
```

Comment:
jakie minusy ma używanie LocalDateTime?

Reply:
Główną wadą może być to, że nie przechowuje informacji o strefie czasowej.